app/Http/Livewire/PublicHome/TodayLecturingSchedules.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Http\Livewire\PublicHome;
+
+use App\Models\LecturingSchedule;
+use Carbon\Carbon;
+use Livewire\Component;
+
+class TodayLecturingSchedules extends Component
+{
+    public $header;
+    public $detailTextButton;
+    public $intervalCarousel;
+    public $lecturingSchedules = [];
+    public $audienceCodes = [];
+    public $linkDetailSchedule;
+    public $isFriday;
```

Comment:
Sebagian besar variable tidak digunakan di view.
```suggestion
    public $lecturingSchedules = [];
```